dataflow/src/org/checkerframework/dataflow/cfg/CFGBuilder.java

```diff
@@ -256,106 +237,53 @@ public static ControlFlowGraph build(
      */
     public static ControlFlowGraph build(
             TreePath bodyPath,
-            ProcessingEnvironment env,
             UnderlyingAST underlyingAST,
             boolean assumeAssertionsEnabled,
-            boolean assumeAssertionsDisabled) {
-        return new CFGBuilder(assumeAssertionsEnabled, assumeAssertionsDisabled)
-                .run(bodyPath, env, underlyingAST);
+            boolean assumeAssertionsDisabled,
+            ProcessingEnvironment env) {
+        TreeBuilder builder = new TreeBuilder(env);
+        AnnotationProvider annotationProvider = new BasicAnnotationProvider();
+        PhaseOneResult phase1result =
+                new CFGTranslationPhaseOne(
+                                builder,
+                                annotationProvider,
+                                assumeAssertionsEnabled,
+                                assumeAssertionsDisabled,
+                                env)
+                        .process(bodyPath, underlyingAST);
+        ControlFlowGraph phase2result = CFGTranslationPhaseTwo.process(phase1result);
+        ControlFlowGraph phase3result = CFGTranslationPhaseThree.process(phase2result);
+        return phase3result;
     }
 
-    /** Build the control flow graph of a method. */
+    // NullAway uses this entry point.
+    @Deprecated
     public static ControlFlowGraph build(
-            CompilationUnitTree root,
+            TreePath bodyPath,
```

Comment:
This method is marked `@Deprecated`, so should the signature stay the same?

Reply:
I think the alignment in the diff is off.
The call is here: https://github.com/uber/NullAway/blob/master/nullaway/src/main/java/com/uber/nullaway/dataflow/DataFlow.java#L110
and the parameters here should match that call.

Reply:
Yes, the diff if weird.  Thanks!